section3.py

```diff
@@ -0,0 +1,26 @@
+import decorator
+
+
+@decorator.decorator
+def change_functionality(func,  *args, **kwargs):
```

Comment:
Section 3 should have been used on another decorator - one that executes a function two times, not on a one that returns surprise